lib/ide/parser/src/jsclient.rs

```diff
@@ -43,6 +45,9 @@ extern "C" {
     #[wasm_bindgen(catch)]
     fn parse
     (input:String, ids:String) -> std::result::Result<String,JsValue>;
+    #[wasm_bindgen(catch)]
+    fn parse_file
```

Comment:
If I see a call like `parse_file("foo")`, I'd tend to assume `"foo"` is path to a file.
I'd name more like `parse_file_contents`. Or even keep this as `parse` and rename the other method to something like `parse_text_representation`.